README.md

```diff
@@ -30,6 +30,8 @@ like:
 
 ![alt tag](./screenshots/Screenshot-2.png)
 
+You can either visit to the website to have hands on experience about its working, from here http://sugarlabs.org/
```

Comment:
Not sure why you chose that URL instead of https://musicblocks.sugarlabs.org . Also, we should be using https